p2p/host/autorelay/options.go

```diff
@@ -10,9 +10,22 @@ import (
 	"github.com/benbjohnson/clock"
 )
 
+// AutoRelay will call this function when it needs new candidates because it is
+// not connected to the desired number of relays or we get disconnected from one
+// of the relays. Implementations must send *at most* numPeers, and close the
+// channel when they don't intend to provide any more peers. AutoRelay will not
+// call the callback again until the channel is closed. Implementations should
+// send new peers, but may send peers they sent before. AutoRelay implements a
+// per-peer backoff (see WithBackoff). See WithMinInterval for setting the
+// minimum interval between calls to the callback. The context.Context passed
+// MAY be canceled when AutoRelay feels satisfied, it will be canceled when the
```

Comment:
This is not a RFC 2119 "may".
```suggestion
// may be canceled when AutoRelay feels satisfied, it will be canceled when the
```